index.js

```diff
@@ -60,6 +60,10 @@ function extract(entry) {
     case "GroupComment":
       // Ignore
       break;
+    case "Junk":
+      console.error("INVALID ENTRY!");
+      console.error(JSON.stringify(entry, null, 2));
+      break;
```

Comment:
Wonder if it makes sense to set the `.exitCode` here as well. So at least the htmllinter would fail w/ a non-zero status code in the case of any invalid syntax.